rules/windows/process_creation/proc_creation_win_susp_copy_lateral_movement.yml

```diff
@@ -54,6 +54,7 @@ detection:
         CommandLine|contains|all:
             - '\\\\'
             - '$'
+            - '\Sysvol\'
```

Comment:
The genreic target is like `\\IP\admin$`
Force to have Sysvol too, breack the logic